src/main/java/net/javadiscord/javabot/systems/qotw/QOTWCloseSubmissionsJob.java

```diff
@@ -1,47 +1,53 @@
 package net.javadiscord.javabot.systems.qotw;
 
+import lombok.RequiredArgsConstructor;
+import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.JDA;
 import net.dv8tion.jda.api.Permission;
-import net.dv8tion.jda.api.entities.*;
-import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.MessageHistory;
 import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
 import net.dv8tion.jda.api.interactions.components.ActionRow;
 import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.requests.restaction.ForumPostAction;
+import net.dv8tion.jda.api.utils.messages.MessageCreateBuilder;
+import net.dv8tion.jda.api.utils.messages.MessageCreateData;
 import net.javadiscord.javabot.data.config.BotConfig;
 import net.javadiscord.javabot.data.config.GuildConfig;
 import net.javadiscord.javabot.data.config.guild.QOTWConfig;
-import net.javadiscord.javabot.data.h2db.DbHelper;
 import net.javadiscord.javabot.systems.notification.NotificationService;
-import net.javadiscord.javabot.systems.qotw.submissions.SubmissionControlsManager;
-import net.javadiscord.javabot.systems.qotw.submissions.dao.QOTWSubmissionRepository;
-import net.javadiscord.javabot.systems.qotw.submissions.model.QOTWSubmission;
-import net.javadiscord.javabot.util.ExceptionLogger;
+import net.javadiscord.javabot.systems.qotw.dao.QuestionQueueRepository;
+import net.javadiscord.javabot.systems.qotw.model.QOTWQuestion;
+import org.jetbrains.annotations.NotNull;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
-import lombok.RequiredArgsConstructor;
-
-import java.sql.Connection;
 import java.sql.SQLException;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+import java.util.stream.Collectors;
 
 /**
  * Job which disables the Submission button.
  */
 @Service
 @RequiredArgsConstructor
 public class QOTWCloseSubmissionsJob {
+	private static final String SUBMISSION_PENDING = "\uD83D\uDD52 ";
+
 	private final JDA jda;
-	private final QOTWPointsService pointsService;
 	private final NotificationService notificationService;
+	private final QuestionQueueRepository questionQueueRepository;
+	private final ExecutorService asyncPool;
 	private final BotConfig botConfig;
-	private final DbHelper dbHelper;
-	private final QOTWSubmissionRepository qotwSubmissionRepository;
 
 	/**
-	 * disable the Submission button.
+	 * Disables the submission button.
+	 *
 	 * @throws SQLException if an SQL error occurs
 	 */
 	@Scheduled(cron = "0 0 21 * * 7")//Sunday 21:00
```

Comment:
It might be best to schedule this earlier so there's more time for reviewing.
Currently, it often happens that nobody reviews the QOTWs.

Reply:
did that